mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -224,7 +227,13 @@ void onClickActivateClient() {
     }
 
     public void inflateClientInformation() {
-        mClientDetailsPresenter.loadClientDetailsAndClientAccounts(clientId);
+        ConnectivityManager connectivityManager = (ConnectivityManager) getContext().getSystemService(Context.CONNECTIVITY_SERVICE);
```

Comment:
Can you separate it into two lines as because of the condition in check style plugin i.e. you can't put more than 100 characters in one line, the build is failing.